packages/contracts/src/constants.ts

```diff
@@ -57,7 +57,10 @@ export const TS_AGGREGATOR_TOKEN_TRANSFER_PROXY_CONTRACT_MAINNET =
 // RFOX on Arbitrum ERC1967Proxy contract address
 // Uncomment me if you want to test RFOX with a shorter cooldown period
 // export const RFOX_PROXY_CONTRACT: Address = '0x1094c4a99fce60e69ffe75849309408f1262d304'
-export const RFOX_PROXY_CONTRACT: Address = '0xac2a4fd70bcd8bab0662960455c363735f0e2b56'
+// export const RFOX_LP_PROXY_CONTRACT: Address = '0x4843373F380aDf80C779cD4dEDC6452E414f7634'
+
+export const RFOX_PROXY_CONTRACT: Address = '0xaC2a4fD70BCD8Bab0662960455c363735f0e2b56'
+export const RFOX_LP_PROXY_CONTRACT: Address = '0x83B51B7605d2E277E03A7D6451B1efc0e5253A2F'
```

Comment:
very nitpick and I think we seen it together, we could change this name to make it more specific and related to the univ2 pool

Reply:
1f47d16ff4b6b4f4d9d29e210450d2ed028c9855